src/migration/get-old-service-agreements-migration.js

```diff
@@ -0,0 +1,133 @@
+import BaseMigration from './base-migration.js';
+import { SERVICE_AGREEMENT_SOURCES } from '../constants/constants.js';
+
+const BATCH_SIZE = 50;
+const GNOSIS_MAINNET_CHAIN_ID = 'gnosis:100';
+const GNOSIS_MAINNET_ASSET_STORAGE_CONTRACT_ADDRESS = '0xf81a8c0008de2dcdb73366cf78f2b178616d11dd';
+
+class GetOldServiceAgreementsMigration extends BaseMigration {
```

Comment:
Would have been good to also update all service agreements lastCommitEpoch to NULL so that nodes will sync and commit again, since asset sync command is broken right now and sharding table has changed a lot in the past few weeks. Nodes would not commit twice anyway as they check if they already committed on submitCommitCommand